satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -377,6 +377,72 @@ read limitoffset (
     orderby desc bucket_usage.rollup_end_time
 )
 
+//--- used serials ---//
+
+model used_serial {
+	key    serial_number
+	unique serial_number storage_node_id
```

Comment:
Depending on our queries it makes sense to revert the order here.
I assume that we query/rollup based on the storagenodeID and not per serial

Reply:
we always query <serial number, storage node id>

Reply:
ah note, we do not roll up this table.... it's just here to 1. figure out bucket_id and 2. to avoid duplicates

Reply:
Oh yeah, than it should be fine.